### PR TITLE
chore: move samples from GoogleCloudPlatform/python-docs-samples

### PR DESCRIPTION
**Pull request by irataxy**
Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:
- [ ] Make sure to open an issue as a [bug/issue](https://github.com/googleapis/python-video-transcoder/issues/new/choose) before writing your code!  That way we can discuss the change, evaluate designs, and agree on the general idea
- [ ] Ensure the tests and linter pass
- [ ] Code coverage does not decrease (if any source code was changed)
- [ ] Appropriate docs were updated (if necessary)

Fixes #<issue_number_goes_here> 🦕

### CODE REVIEW

**Comment by parthea** [comment]
Hi @nicain , Please could you review/approve?

**Review by nicain — APPROVED** [review]
LGTM. Great work!

File: README.rst [hunk 1]
```diff
@@ -68,9 +68,11 @@ Windows
     <your-env>\Scripts\activate
     <your-env>\Scripts\pip.exe install google-cloud-video-transcoder
 
+
 Next Steps
 ~~~~~~~~~~
 
+-  See the [Samples](./samples/snippets/README.md).
```
Comment (dandhlee) [inline README.rst:75]: Nit: this link doesn't seem to show up properly, but feel free to address this in a different PR.

**Review by busunkim96 — APPROVED** [review]

**Comment by busunkim96** [comment]
Thanks for moving these here! Please make sure to adjust any references to the samples in the docs published to cloud.google.com before deleting the ones in python-docs-samples.

**Pull request merged**